deploy/modules/file.py

```diff
@@ -1,38 +1,17 @@
-import datetime
-from typing import Dict, List
+from viur.core import i18n, db, current
+from viur.core.modules.file import File
 
-from viur.core import db, utils
-from viur.core.modules.file import File as _File
 
+class File(File):
 
-class File(_File):
+    def getAvailableRootNodes(self, *args, **kwargs):
+        # Any user who is logged in can see the root-node.
+        if current.user.get():
+            repository = self.ensureOwnModuleRootNode()
 
-    def getAvailableRootNodes(self, *args, **kwargs) -> List[Dict]:
-        if utils.getCurrentUser():
-            repo: db.Entity = self.ensureOwnModuleRootNode()
-
-            res = [
-                {
-                    "name": "Files",
-                    "key": repo.key
-                }
-            ]
-            return res
+            return [{
+                "name": i18n.translate("Files"),
+                "key": repository.key
+            }]
 
         return []
-
-    def ensureOwnModuleRootNode(self) -> db.Entity:
-        """
-        Ensures, that general root-node for the current module exists.
-        If no root-node exists yet, it will be created.
-
-        :returns: The entity of the root-node.
-        """
-        key = "rep_module_repo"
-        kindName = self.viewSkel("node").kindName
-
-        return db.GetOrInsert(
-            db.Key(kindName, key),
-            creationdate=datetime.datetime.now(),
-            rootNode=1
-        )
```

Comment:
Why did you remove this method? It's still called by `getAvailableRootNodes`.

Reply:
It is defined in the `Tree`-prototype and does exactly the same.

Reply:
Ahh, okay it's in `Tree`, I just searched for it in the core `File` module.